client/performance-profiler/components/metrics-insight/insight-header.tsx

```diff
@@ -38,6 +38,9 @@ export const InsightHeader: React.FC< InsightHeaderProps > = ( props ) => {
 					<span className={ clsx( 'value', { [ type ]: true } ) }> { value }</span>
 				</span>
 			) }
+			{ type === 'fail' && (
+				<span className={ clsx( 'impact fail', { 'is-mobile': isMobile() } ) }>High Impact</span>
+			) }
```

Comment:
Is this how we define if we should have a `high impact` badge? I understood it was a different check,

Reply:
We should show an audit as `High Impact` when the `metricSavings` key is not empty.

Reply:
Oh sorry, I missed that this should be the trigger for it. I've added it in ce4f4785c7